src/pages/categories/[slug].js

```diff
@@ -1,16 +1,26 @@
 import { getAllAuthors, getUserBySlug } from 'lib/users';
 import { getAllCategories, getCategoryBySlug } from 'lib/categories';
 import { getPostsByCategoryId } from 'lib/posts';
+import { WebpageJsonLd } from 'lib/json-ld';
 
 import TemplateArchive from 'templates/archive';
 import Title from 'components/Title';
 
+import useSite from 'hooks/use-site';
+
 import styles from 'styles/pages/Post.module.scss';
 
 export default function Category({ category, posts }) {
   const { name, description } = category;
-
-  return <TemplateArchive title={name} Title={<Title title={name} />} description={description} posts={posts} />;
+  const { metadata = {} } = useSite();
+  const { title: siteTitle } = metadata;
+
+  return (
+    <>
+      <WebpageJsonLd title={name} description={description} siteTitle={siteTitle} />
```

Comment:
did you try adding this to the `<TemplateArchive` instead of here so that all the pages could benefit from it?

Reply:
It's true! I don't know why I was thinking couldnt reuse it...